src/serac/physics/integrators/displacement_hyperelastic_integrator.cpp

```diff
@@ -31,6 +31,13 @@ void DisplacementHyperelasticIntegrator::CalcKinematics(const mfem::FiniteElemen
 
   solid_util::calcDeformationGradient(du_dX_, F_);
 
+  if (thermal_material_) {
+    F_copy_ = F_;
+    F_thermal_.SetSize(du_dX_.Width());
+    thermal_material_->evalThermalDeformationGradient(F_thermal_);
+    mfem::Mult(F_thermal_, F_copy_, F_);
```

Comment:
I haven't done enough finite thermoelasticty to say which is more common, but in the META write up, we have the terms of the multiplicative decomposition in opposite order: F = F_elastic * F_thermal. In every case I can think of, F_thermal is going to be diagonal, so this is a distinction without a difference, but I thought I would mention it (and possibly update the write up if necessary).

Reply:
Yep - this is a typo. Thanks for catching it!